src/app/view/components/_common/dot-page-selector/dot-page-selector.component.scss

```diff
@@ -9,13 +10,14 @@
         .ui-autocomplete-list-item {
             white-space: nowrap;
         }
+
+        .autocomplete-input {
+            padding-right: $basic-padding * 4;
```

Comment:
Why don't we create a global class that we can put in any input to do this? so in case we have yo use the helper in another input, we just use that clase.